lib/entities/quote/DutchQuoteFactory.ts

```diff
@@ -125,7 +125,8 @@ export class DutchQuoteFactory {
 
     const classicAmounts = DutchQuote.applyGasAdjustment(
       { amountIn: classic.amountInGasAdjusted, amountOut: classic.amountOutGasAdjusted },
-      classic
+      classic,
+      quote.request.config.gasAdjustmentBps
```

Comment:
Would love to be able to unit test this piping directly, but the dependency is not inject - testing indirectly by comparing end amount

Reply:
Commented below on this, but it looks like `quote.request.config.gasAdjustmentBps` is not making it here! Looking into it

Reply:
LMK if I can help.. it looks like it works for me when I run it locally.